weed/shell/command_volume_fsck.go

```diff
@@ -654,9 +658,10 @@ func (c *commandVolumeFsck) collectVolumeIds() (volumeIdToServer map[string]map[
 					isEcVolume: true,
 					isReadOnly: true,
 				}
+				volumeCount += 1
```

Comment:
Shard count is different from volume count.

Reply:
Sorry, I don't have much information of seaweedfs.
I thought the count here is volume count. normal volume + ec volume (not shard)
Moreover, the reported value is not changed after code changes. (I have ec volumes too in my cluster)

Please let me know how can I fix this.

Reply:
you can have separate counts for normal volumes and for ec shards.

This PR adds `count` but only for writing to verbose output. Seems there are no actual changes?